.circleci/config.base.yml

```diff
@@ -597,10 +597,21 @@ jobs:
       - restore_cache:
           key: amplfiy-pkg-tag-{{ .Branch }}-{{ .Revision }}
       - run:
-          name: Publish Amplify CLI GitHub release
+          name: Update 'main' branch to match release
           command: |
+            commit=$(git rev-parse HEAD)
             version=$(cat .amplify-pkg-version)
-            yarn ts-node scripts/github-release.ts $version
+            git push origin release:main -f
```

Comment:
hm, yeah if CI force pushes in the middle of an incident, we may have an issue. maybe this step should be manual for now.

Reply:
I thought about this and started to write a script, but I think to get the full benefit this step should be automated so that the on-call doesn't have to go back and update their remote and checkout the tag again. I can add `--force-with-lease` and then I will follow up with a manual script if for some reason that fails